### DISCUSSION TITLE
Hosting NSFW content on a github/github page

### DISCUSSION THREAD

**Discussion by questianon** [community/community#37695] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi, I own a public repository aimed at the documentation of Stable Diffusion. As such, people have discovered and created both SFW and NSFW content that I wish to document for everyone to use. Would hosting this content and displaying images of training samples/documentation (which may be NSFW) on the repo/a related github page be against Github's TOS? Thank you in advance.

**Comment by is-it-ayush** [comment, marked as answer]
@questianon According to [Github Docs](https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages), 
>In addition, your use of GitHub Pages is subject to the [GitHub Terms of Service](https://docs.github.com/en/free-pro-team@latest/github/site-policy/github-terms-of-service), including the restrictions on get-rich-quick schemes, sexually obscene content, and violent or threatening content or activity.

If I were in your shoes, I'd look for a more stable solution such as Gitlab etc. and host my own repo/docs(your case) there to be on the safer end and not be afraid of unotified removal's. :)

**Comment by airtower-luna** [comment]
It really depends on what exactly. "Sexually obscene content" isn't allowed, but that's not the same as *anything* NSFW or related to sexuality. [The policy on the matter](https://docs.github.com/en/site-policy/acceptable-use-policies/github-sexually-obscene-content) also says:

> We recognize that not all nudity or content related to sexuality is obscene. We may allow visual and/or textual depictions in artistic, educational, historical or journalistic contexts, or as it relates to victim advocacy.

**Comment by AneeVerse** [comment]
I think, click the gayer icon in to top right corner and then see the page section, this might help
